### PR TITLE
added queries

### PR DESCRIPTION
**Pull request by xingcbrew**
@joebrew, I've added 3 rows to `clean_database_fixes.py` for you to review. They were new queries (ie I couldn't copy/paste from previous rows), so I had to guess at how to write them. The first two rows share one correction id, but it looked like 2 changes were needed based on the response and I didn't know if I could write them in one query.

### CODE REVIEW

File: scripts/clean_database_fixes.py [hunk 1]
```diff
@@ -208,6 +208,9 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
 implement(id = 'no_va_id_9a3207b7-6925-4441-b834-723ae93af283', query = "UPDATE clean_va SET death_id='ZVB-263-701' WHERE instance_id='9a3207b7-6925-4441-b834-723ae93af283'")
 implement(id = 'no_va_id_c6a098bb-55f8-4160-a6e5-5f2afbe9082e', query = "UPDATE clean_va SET death_id='ZVB-286-701' WHERE instance_id='c6a098bb-55f8-4160-a6e5-5f2afbe9082e'")
 implement(id = 'no_va_id_f754caea-0a7d-42c9-a6af-a52d18a1e8ae', query = "UPDATE clean_va SET death_id='EDU-196-701' WHERE instance_id='f754caea-0a7d-42c9-a6af-a52d18a1e8ae'")
+implement(id = 'strange_hh_code_00f5b8ba-739c-428d-ae19-27f2be92044e', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO' WHERE instance_id='00f5b8ba-739c-428d-ae19-27f2be92044e'")
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:211]: @xingcbrew a change to the hamlet code can have a lot of downstream changes (ie, it's a pretty major change). That is, it effects:

*clean_minicensus_main*
-`hh_hamlet_code`
-`hh_hamlet`
-`hh_village`
-`hh_id`

*clean_minicensus_people*
-`pid`
`permid`

So, for each of these kinds of changes, you should first:
1. Have a look at the data for the `instance_id` in question
2. Make changes to any "downstream" effected variables.


In the case of instance_id `00f5b8ba-739c-428d-ae19-27f2be92044e`, it looks like the people for that household were correctly given IDs (`DEO-`). So, in this particular case, you don't have to make any changes to the `people` table.

However, it looks like the variables in `clean_minicensus_main` where not correct:
-`hh_hamlet_code`
-`hh_hamlet`
-`hh_village`
(`hh_id` was correct)

Therefore, the `query` part of your line should be longer. The following:

implement(id = 'strange_hh_code_00f5b8ba-739c-428d-ae19-27f2be92044e', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma', hh_hamlet='4 de Outubro' WHERE instance_id='00f5b8ba-739c-428d-ae19-27f2be92044e'")

(Note that I had to retrieve the hamlet and village name from the locations hierarchy: https://docs.google.com/spreadsheets/d/1hQWeHHmDMfojs5gjnCnPqhBhiOeqKWG32xzLQgj5iBY/edit#gid=1134589765)

File: scripts/clean_database_fixes.py [hunk 2]
```diff
@@ -208,6 +208,9 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_hh_code_00f5b8ba-739c-428d-ae19-27f2be92044e', query = "UPDATE clean_minicensus_main SET hh_hamlet='4 de Outubro' WHERE instance_id='00f5b8ba-739c-428d-ae19-27f2be92044e'")
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:212]: The `id` of the anomaly here is the same as the previous one. Each anomaly should be resolved y only one `implement` call. Therefore, the changing of the `hh_hamlet` value (what you've done here) should instead be done above, and this line should be removed.

File: scripts/clean_database_fixes.py [hunk 3]
```diff
@@ -208,6 +208,9 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_wid_enumerations_2939b05a-3bbe-4c1b-81fe-6eac54d47dc9', query = "UPDATE clean_enumerations SET wid='427' WHERE instance_id='2939b05a-3bbe-4c1b-81fe-6eac54d47dc9'")
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:213]: The above is good.

**Review by joebrew — CHANGES_REQUESTED** [review]
@xingcbrew Added some comments. Back to you.

**Comment by xingcbrew** [comment]
@joebrew I've updated the correction code based on your comments and completed a few more rows of corrections.

File: scripts/clean_database_fixes.py [hunk 4]
```diff
@@ -208,6 +208,36 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
 implement(id = 'no_va_id_9a3207b7-6925-4441-b834-723ae93af283', query = "UPDATE clean_va SET death_id='ZVB-263-701' WHERE instance_id='9a3207b7-6925-4441-b834-723ae93af283'")
 implement(id = 'no_va_id_c6a098bb-55f8-4160-a6e5-5f2afbe9082e', query = "UPDATE clean_va SET death_id='ZVB-286-701' WHERE instance_id='c6a098bb-55f8-4160-a6e5-5f2afbe9082e'")
 implement(id = 'no_va_id_f754caea-0a7d-42c9-a6af-a52d18a1e8ae', query = "UPDATE clean_va SET death_id='EDU-196-701' WHERE instance_id='f754caea-0a7d-42c9-a6af-a52d18a1e8ae'")
+implement(id = 'strange_hh_code_00f5b8ba-739c-428d-ae19-27f2be92044e', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma', hh_hamlet='4 de Outubro' WHERE instance_id='00f5b8ba-739c-428d-ae19-27f2be92044e'")
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:211]: @xingcbrew Have you confirmed that for all of these `strange_hh_code_` anomalies the data in the `people` table is correct? That is, in many of these, you are making changes only to the `minicensus_main` table, but no changes to `mini_census_people`. If the hamlet code was incorrect, I would expect that both the `pid` and `permid` fields of the `people` table are also incorrect...

File: scripts/clean_database_fixes.py [hunk 5]
```diff
@@ -208,6 +208,36 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_hh_code_0f15fdb4-3b7a-41a3-8fa4-1abe03849212', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma', hh_hamlet='Mifarinha', hh_ward='Mopeia sede | Cuacua' WHERE instance_id='0f15fdb4-3b7a-41a3-8fa4-1abe03849212'")
+implement(id = 'strange_hh_code_1b6a9ce2-8d36-4136-869d-a117f0192449', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='AMB', hh_village='Chamanga', hh_hamlet='Ambrosio' WHERE instance_id='1b6a9ce2-8d36-4136-869d-a117f0192449'")
+implement(id = 'strange_hh_code_2624afc1-f262-4bda-b20f-34176ed1c13b', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma, hh_hamlet='Mifarinha' WHERE instance_id='2624afc1-f262-4bda-b20f-34176ed1c13b'")
+implement(id = 'strange_hh_code_287b28f8-c6ba-4b4e-bd6f-b6778a729d93', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma, hh_hamlet='4 de Outubro', hh_id='DEO-033' WHERE instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'"; 
+    "UPDATE clean_minicensus_people SET pid='DEO-033-001' WHERE num='1' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:216]: Changes should be made both to `pid` and `permid` (they're identical, but both need to be updated...)

File: scripts/clean_database_fixes.py [hunk 6]
```diff
@@ -208,6 +208,36 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+    "UPDATE clean_minicensus_people SET pid='DEO-033-002' WHERE num='2' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-003' WHERE num='3' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-004' WHERE num='4' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-005' WHERE num='5' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-006' WHERE num='6' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-007' WHERE num='7' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'")
+implement(id = 'strange_hh_code_2c54e783-2281-429f-9896-060261a148db', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma, hh_hamlet='Mifarinha' WHERE instance_id='2c54e783-2281-429f-9896-060261a148db'")
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:223]: Same question for the next 10 lines or so. Do we need to update more than just the hamlet code in the `clean_minicensus_main` table? What about the `pid` and `permid` fields in the `clean_minicensus_people` table with the same instance id?

**Review by joebrew — CHANGES_REQUESTED** [review]
@xingcbrew Back to you

File: scripts/clean_database_fixes.py (hunk 5 above)
Comment (xingcbrew) [reply scripts/clean_database_fixes.py]: @joebrew Right, I've corrected this now. Thanks!

**Comment by joebrew** [comment]
@xingcbrew As discussed, please:

1. Add comment lines (beginning with #) to indicate that a seemingly incomplete fix is actually complete (ie, weird hamlet codes which get changed, but no changes made to the members of the household)

2. Add a `who = 'Xing Brew'` argument to all of the lines you wrote.

File: scripts/clean_database_fixes.py [hunk 7]
```diff
@@ -208,7 +208,39 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_hh_code_00f5b8ba-739c-428d-ae19-27f2be92044e', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma', hh_hamlet='4 de Outubro' WHERE instance_id='00f5b8ba-739c-428d-ae19-27f2be92044e'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_0f15fdb4-3b7a-41a3-8fa4-1abe03849212', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma', hh_hamlet='Mifarinha', hh_ward='Mopeia sede | Cuacua' WHERE instance_id='0f15fdb4-3b7a-41a3-8fa4-1abe03849212'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_1b6a9ce2-8d36-4136-869d-a117f0192449', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='AMB', hh_village='Chamanga', hh_hamlet='Ambrosio' WHERE instance_id='1b6a9ce2-8d36-4136-869d-a117f0192449'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_2624afc1-f262-4bda-b20f-34176ed1c13b', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma, hh_hamlet='Mifarinha' WHERE instance_id='2624afc1-f262-4bda-b20f-34176ed1c13b'", who = 'Xing Brew')
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:215]: There is a missing ' after the word `Marruma`, which causes this to fail.

File: scripts/clean_database_fixes.py [hunk 8]
```diff
@@ -208,7 +208,39 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_hh_code_287b28f8-c6ba-4b4e-bd6f-b6778a729d93', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma, hh_hamlet='4 de Outubro', hh_id='DEO-033' WHERE instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:216]: Another missing '

File: scripts/clean_database_fixes.py [hunk 9]
```diff
@@ -208,7 +208,39 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+implement(id = 'strange_hh_code_1b6a9ce2-8d36-4136-869d-a117f0192449', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='AMB', hh_village='Chamanga', hh_hamlet='Ambrosio' WHERE instance_id='1b6a9ce2-8d36-4136-869d-a117f0192449'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_2624afc1-f262-4bda-b20f-34176ed1c13b', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma, hh_hamlet='Mifarinha' WHERE instance_id='2624afc1-f262-4bda-b20f-34176ed1c13b'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_287b28f8-c6ba-4b4e-bd6f-b6778a729d93', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='DEO', hh_village='Marruma, hh_hamlet='4 de Outubro', hh_id='DEO-033' WHERE instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'"; 
+    "UPDATE clean_minicensus_people SET pid='DEO-033-001', permid='DEO-033-001' WHERE num='1' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:217]: For these multiple statements, EVERYTHING needs to go between the ".

What you've done is to make multiple statements wrapped in "", separated by semicolons. That is, you've done:
"statement a"; "statement b"; "statement c";

Instead it needs to be "statement a; statement b; statement c"

File: scripts/clean_database_fixes.py [hunk 10]
```diff
@@ -208,7 +208,39 @@ def implement(id, query = '', who = 'Joe Brew', cur = cur, dbconn = dbconn):
...
+    "UPDATE clean_minicensus_people SET pid='DEO-033-002', permid='DEO-033-002' WHERE num='2' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-003', permid='DEO-033-003' WHERE num='3' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-004', permid='DEO-033-004' WHERE num='4' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-005', permid='DEO-033-005' WHERE num='5' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-006', permid='DEO-033-006' WHERE num='6' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'";
+    "UPDATE clean_minicensus_people SET pid='DEO-033-007', permid='DEO-033-007' WHERE num='7' and instance_id='287b28f8-c6ba-4b4e-bd6f-b6778a729d93'", who = 'Xing Brew')
+implement(id = 'strange_hh_code_2c54e783-2281-429f-9896-060261a148db', query = "UPDATE clean_minicensus_main SET hh_hamlet_code='MIF', hh_village='Marruma, hh_hamlet='Mifarinha' WHERE instance_id='2c54e783-2281-429f-9896-060261a148db'", who = 'Xing Brew')
```
Comment (joebrew) [inline scripts/clean_database_fixes.py:224]: Lots more of these missing ' after Marruma. Check quotations on all strings...

**Review by joebrew — COMMENTED** [review]
@xingcbrew A few comments in line. Back to you...

**Review by joebrew — APPROVED** [review]
@xingcbrew Looks good. I'm going to go ahead and merge.

**Pull request merged**